extension/src/frontend/view/ToolBox.tsx

```diff
@@ -0,0 +1,71 @@
+import { css } from '@emotion/react'
```

Comment:
Moved, modified to use `@/components/primitives/Table` and added the event drawer toggle.